src/inline-assembly.md

```diff
@@ -191,11 +248,10 @@ Here is the list of currently supported register classes:
 
 > **Notes**:
 > - On x86 we treat `reg_byte` differently from `reg` because the compiler can allocate `al` and `ah` separately whereas `reg` reserves the whole register.
->
```

Comment:
Out of curiosity, what led you to remove these lines?

Reply:
Originally, I had added identifiers on each of the bullet points in this note, but then decided against it.  The lines were removed as part of that.  I did then later notice and thought about putting these newlines back in, but other similar "notes" sections in the document use the condensed formatting (this was the only exception), and it felt not entirely unrelated to make this condensed, at that formatting seemed to fit better with the idea that this is all part of one thing in one identified section, so I decided to leave it.